main.go

```diff
@@ -0,0 +1,139 @@
+package main
+
+import (
+	"bufio"
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+	"sort"
+)
+
+type Record struct {
+	ID        uint
+	FirstName string
+	LastName  string
+	PhoneN    string
+}
+
+func removeEntry() (errr string) {
+	var Records []Record
```

Comment:
Some small remarks:
- Try to name local variables starting with lower case. Capital letters are inconsistent, and they have special meaning in Go.
-  Try to declare variables as close as possible to the place they are used. In this case I'd recommend to declare it right before the `Decode` call